processes/third_party_vuln_process.md

```diff
@@ -0,0 +1,86 @@
+This document describes the management of vulnerabilities within the Node.js
+ecosystem. Vulnerabilities in Node.js core are out of this scope.
+
+# Definitions
+
+* package: in this document, a package is a module available widely to use 
+with Node.js. Most modules are hosted on the npmjs.org repository. 
+
+# Process
+
+Individuals who find potential vulnerabilities in a package are invited 
+to complete a vulnerability report on the dedicated HackerOne organization:
+
```

Comment:
It should also be possible for a vulnerability to be reported via email (which can be forwarded to HackerOne and will auto-create a vuln report).

Reply:
I have an issue on setting up this on H1 side right now. I have pinged H1 on that for assistance.